docs/decisions/0002-feature-based-application-organization.rst

```diff
@@ -0,0 +1,213 @@
+2. Feature-based Application Organization
```

Comment:
This decision is taken from the template app and not need to be here.

Reply:
@idegtiarov this is fixed in https://github.com/edx/frontend-lib-special-exams/pull/2 already, once merged
will update it here also